domain-cc/README.md

```diff
@@ -33,3 +52,13 @@ pip install -r dev-requirements.txt
 # MAKE SURE you are in adb-vro/domain-cc to get the right pre-commit-config.yaml installed
 pre-commit install
 ```
+
+## Building docs
+```
+source ~/.virtualenvs/domain-cc/bin/activate
+cd src/
+uvicorn api:app --reload &
+cd ../..
+cd app/src/main/java/gov/va/vro/config 
+curl localhost:8000/openapi.json > domain-cc-openapi.json
```

Comment:
To make it clear that this service is not intended to be exposed when VRO is deployed, please use an obviously-internal-and-for-development-only port:
```suggestion
curl localhost:18000/openapi.json > domain-cc-openapi.json
```
I'll document that shortly.